stable/gangway/templates/deployment.yaml

```diff
@@ -52,6 +52,9 @@ spec:
             mountPath: /etc/gangway/tls
             readOnly: true
         {{ end }}
+{{- if ne (len .Values.extraVolumeMounts) 0 }}
+{{ toYaml .Values.extraVolumeMounts | trim | indent 10 }}
```

Comment:
Yeah, the entire container-block is indented like,
```
      containers:
        - name: {{ .Chart.Name }}
```
and not like
```
      containers:
      - name: {{ .Chart.Name }}
```
So it's 10 instead of 8. (Same for the 8 and not 6 on the extraVolumes.)

Reply:
Align entire chart to use compact-style block sequences in a2759b8.